google/resource-snippets/pubsub-v1/pubsub.jinja

```diff
@@ -14,48 +14,30 @@
 
 resources:
   - type: pubsub.v1.topic
-    name: {{ properties["topic_name"] }}
+    name: {{ properties["topicName"] }}
     properties:
       topic: my-pubsub-topic
```

Comment:
This should be `{{ properties["topicName"] }}` too, since this will be the actual topic name whereas the name above is more like the DM resource name. We reference objects using the DM resource names i.e. (`$(ref.{{ properties["topicName"] }}`). I'd imagine you would want to make the actual topic name configurable.

Reply:
Sorry, where do you see topic_name? I don't see it in my jinja file, I see the correct "topicName"

Reply:
I meant instead of `my-pubsub-topic`, it should be `{{ properties["topicName"] }}`. Because users of the template would want to configure the actual topic name, and this `topic` is what controls that.

i.e. I'd expect setting `topicName: my-very-unique-topic-name` in the YAML file to reflect that in the actual topic created, but this config doesn't do that.